src/ext.js

```diff
@@ -33,164 +33,175 @@ export default function ext({ translator, shouldHide, senseNavigation, theme, is
+                        const fields = await hyperCubeHandler.app.getFieldList();
+                        return fields.map((field) => ({
+                          label: field.qName,
+                          value: field.qName,
+                        }));
+                      },
+                      show: (data) => checkShowAction(data, "field"),
+                    },
+                    variable: {
+                      type: "string",
+                      ref: "variable",
+                      component: "expression-with-dropdown",
+                      translation: "Common.Variable",
+                      defaultValue: "",
+                      expressionType: "StringExpression",
+                      options: async (action, hyperCubeHandler) => {
+                        const variables = await hyperCubeHandler.app.getVariableList();
+                        return variables
+                          .filter((v) => !v.qIsReserved || (v.qIsReserved && action.showSystemVariables))
+                          .map((v) => ({
+                            label: v.qName,
+                            value: v.qName,
+                          }));
+                      },
+                      show: (data) => checkShowAction(data, "variable"),
+                    },
+                    showSystemVariables: {
+                      type: "boolean",
+                      ref: "showSystemVariables",
+                      translation: "ExpressionEditor.SystemVariables",
+                      defaultValue: false,
+                      show: (data) => checkShowAction(data, "variable"),
+                    },
+                    softLock: {
+                      type: "boolean",
+                      ref: "softLock",
+                      translation: "Object.ActionButton.Softlock",
+                      defaultValue: false,
+                      show: (data) => checkShowAction(data, "softLock"),
+                    },
+                    value: {
+                      type: "string",
+                      ref: "value",
+                      component: "string",
+                      translation: "properties.value",
+                      expression: "optional",
+                      show: (data) => checkShowAction(data, "value"),
+                    },
+                    partial: {
+                      type: "boolean",
+                      ref: "partial",
+                      translation: "Object.ActionButton.Partial",
+                      defaultValue: false,
+                      show: (data) => checkShowAction(data, "partial"),
+                    },
+                    automationProps: {
+                      type: "items",
+                      grouped: false,
+                      items: getAutomationProps(multiUserAutomation, getAutomations),
+                      show: (data) => checkShowAction(data, "automation"),
+                    },
+                    cyclicGroup: {
```

Comment:
this is just moved right?

Reply:
Yes. Needed to add the toggle on the same level as the actions. So this has all moved down a bit:

![image](https://github.com/qlik-oss/sn-action-button/assets/25658598/68a56ddb-60dd-4115-ba2b-c0c52a291a3c)